packaging/Microsoft.NET.Sdk.WindowsDesktop/Microsoft.NET.Sdk.WindowsDesktop.ArchNeutral.csproj

```diff
@@ -24,5 +24,8 @@
     <PackagingContent Include="Sdk\*" SubFolder="root\Sdk" />
     <PackagingContent Include="targets\*" SubFolder="root\targets" />
     <PackagingContent Include="useSharedDesignerContext.txt" SubFolder="root" />
+
+    <!-- Windows Forms specific -->
+    <PackagingContent Include="$(PkgMicrosoft_Private_Winforms)\Wpf.Sdk.WindowsDesktop\*" SubFolder="root\targets" />
```

Comment:
Agree that its supposed to be as simple as this. IIRC WPF repo uses  `GeneratePathProperty` with most Nuget packages and its going to be true for the WinForms transport package as well. 

Side note, I've always been sad that WinForms didn't get the casing for the casing for the transport package name like `WinForms` and it's like `Winforms` - so it's pretty confusing to type. 

Also, don't bundle your targets inside a subfolder named "Wpf.Sdk.WindowsDesktop" - bundle it inside a subfolder named "build\" or something canonical that NuGet understands well. See https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package. (Not a blocking comment - what you have here ought to work just fine). 

Overall your approach looks fine to me.

Reply:
Outside these props, ensure that when `PkgMicrosoft_Private_Winforms` is undefined, the build fails. Good chance that's already the case (or should be) but it's very important this happens from here onwards. 

Likewise, make sure that the build fails if `S.W.F.Analyser.[props|targets]` aren't found under the expected locations under `$(PkgMicrosoft_Private_Winforms)`. 

The way `PackagingContent` works right now, if those files aren't found, then packaging will just proceed along without them - so you're responsible for validating.

Reply:
> IIRC WPF repo uses `GeneratePathProperty` with most NuGet packages and its going to be true for the WinForms transport package as well.

It is here:
https://github.com/dotnet/wpf/blob/89d172db0b7a192de720c6cfba5e28a1e7d46123/eng/WpfArcadeSdk/tools/SdkReferences.targets#L3-L10


> Also, don't bundle your targets inside a subfolder named "Wpf.Sdk.WindowsDesktop" - bundle it inside a subfolder named "build" or something canonical that NuGet understands well. See [docs.microsoft.com/en-us/nuget/create-packages/creating-a-package](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package). (Not a blocking comment - what you have here ought to work just fine).

We have some props/targets that go to WPF and some that go to WindowsDesktop. This is an attempt to provide some indication of the destination (as well as some docs in our repo).
Also I tried putting these under "build", and that results in [NuGet Warning NU5129](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5129).



> Outside these props, ensure that when `PkgMicrosoft_Private_Winforms` is undefined, the build fails.....

Good idea 👍 

Thank you

Reply:
Added a target to verify
![image](https://user-images.githubusercontent.com/4403806/120473477-66d60e00-c3ea-11eb-8a0e-72fd4b991de7.png)

Reply:
NU5129 seems like a common enough problem and I've had to suppress it elsewhere for similar reasons when creating NuGet packages with build\ subfolder.

Reply:
Our transport package has the following structure:
![image](https://user-images.githubusercontent.com/4403806/120605776-cf2bfa80-c491-11eb-9a4c-6dc137c95cbd.png)